pymeasure/instruments/novanta/fpu60.py

```diff
@@ -0,0 +1,142 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2023 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+import re
+
+from pymeasure.instruments import Instrument
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class Fpu60(Instrument):
+    """Represents a fpu60 power supply unit for the finesse laser series by Laserquantum,
+    a Novanta company.
+
+    The instrument responds to every command sent.
+    """
+
+    def __init__(self, adapter, name="Laserquantum fpu60 power supply unit", **kwargs):
+        super().__init__(adapter,
+                         name=name,
+                         includeSCPI=False,
+                         asrl={'baud_rate': 19200},
+                         write_termination="\r",
+                         read_termination="\r\n",
+                         **kwargs)
+
+    interlock_enabled = Instrument.measurement(
+        "INTERLOCK?",
+        """Measure the interlock enabled status (bool).""",
```

Comment:
Minor point: is it mandatory to use "measure" word in docstring even if the value is a flag presumably not sampled or measured. I have seen also below you used the word "get" for serial number.

Reply:
That was from the previous documentation, which stated to have "Measure" as the docstring name. I change it.

Reply:
```suggestion
        """Get the interlock enabled status (bool).""",
```